plugins/modules/swim_intent.py

```diff
@@ -847,9 +862,9 @@ def get_diff_import(self):
             if image_exist:
                 image_id = self.get_image_id(name)
                 self.have["imported_image_id"] = image_id
-                self.msg = "Image {0} already exists in the Cisco DNA Center".format(name)
+                self.msg = "Image {0} already exists in the Cisco Catalyst Center".format(name)
```

Comment:
self.message = "Image '{0}' already exists in the Cisco Catalyst Center".format(name)

Reply:
updated.